README.MD

```diff
@@ -88,7 +88,9 @@ Contributions welcome. Add links through pull requests or create an issue to sta
 - [MyFitnessPal](http://www.myfitnesspal.com/) - Food tracking and diet plan app (iOS & Android).
 - [Fat Secret](https://www.fatsecret.com/) - Calorie counter and diet tracker for weight loss (iOS & Android).
 - [Calorie Counter](https://www.caloriecount.com/) - Nutritional database and calorie counting app (iOS & Android).
+- [Cronometer](https://cronometer.com/) - Food, activity, and biometric tracker (iOS & Android).
 - [Zero](https://itunes.apple.com/us/app/zero-fasting-tracker/id1168348542?mt=8) - A simple fasting tracker used for intermittent, circadian rhythm, and custom fasting (iOS). 
+- [Vora](https://play.google.com/store/apps/details?id=com.fastapp&hl=en_US) Fasting tracker (Android).
```

Comment:
Don't we have a better link than the app store? Also, need a hyphen.

Reply:
Good suggestion. Added missing hyphen and found a better URL. I also found a similar cross-platform landing page for Zero just now, so I've included that update in this PR too.